src/users/containers/utils/user-roles-util.ts

```diff
@@ -36,14 +38,16 @@ export class UserRolesUtil {
     return roles;
   }
 
-  public static mapEditUserRoles(user: any, rolesAdd: any[], rolesDelete: any[]): any {
+  public static mapEditUserRoles(user: any, userId: string, rolesAdd: any[], rolesDelete: any[], accessTypes: UserAccessType[] = []): EditUserModel {
```

Comment:
do you still need user id if the user object is being provided?

Reply:
userId param removed. Tested both new and old edit user functions and both work using the id within the user object